libs/crypto/src/lib.rs

```diff
@@ -84,31 +116,37 @@ impl fmt::Debug for DbPasswordV1 {
 #[derive(Debug)]
 pub struct CryptoPolicy {
     pub(crate) pbkdf2_cost: usize,
+    // https://docs.rs/argon2/0.5.0/argon2/struct.Params.html
+    // defaults to 19mb memory, 2 iterations and 1 thread, with a 32byte output.
+    pub(crate) argon2id_params: Params,
 }
 
 impl CryptoPolicy {
     pub fn minimum() -> Self {
         CryptoPolicy {
             pbkdf2_cost: PBKDF2_MIN_NIST_COST,
+            argon2id_params: Params::default(),
         }
     }
 
-    pub fn time_target(t: Duration) -> Self {
-        let r = match Password::bench_pbkdf2(PBKDF2_MIN_NIST_COST * 10) {
+    pub fn time_target(target_time: Duration) -> Self {
+        const PBKDF2_BENCH_FACTOR: usize = 10;
```

Comment:
Wouldn't this const be better up with the other things? 😄

Reply:
It could be, but it's also only ever used in this one function so I think tho locality is nicer